src/sentry/sentry_metrics/indexer/base.py

```diff
@@ -189,6 +252,107 @@ def __getitem__(self, org_id: int) -> Mapping[str, Optional[int]]:
         return self.results[org_id]
 
 
+class UseCaseKeyResults:
```

Comment:
Can you please add docstrings for all these classes? I'm not really sure how it's supposed to be used